### PR TITLE
[DEVX-1807] Add saucectl docs for testApp on suite level

### PR DESCRIPTION
**Pull request by tianfeng92**
Add `testApp` settings on suite level.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md [hunk 1]
```diff
@@ -542,6 +542,16 @@ The name of the test suite, which will be reflected in the results and related a
 ```
 ---
 
+### `testApp`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Set test application on suite level. See the full [usage](#testapp). If this field is not set, saucectl will pick the default `testApp` from [espresso](#espresso) level.
+
+```yaml
+  testApp: ./apps/calc-success.apk
```
Comment (alexplischke) [inline docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:551]: Would be good to put the parent context (suite) in here as well. Makes it easier to see where this attribute belongs. If you recall, the idea was to make those changes incrementally, which should make the docs easier to use as reference material :)

Comment (tianfeng92) [reply docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:551]: make sense.

File: docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md [hunk 2]
```diff
@@ -542,6 +542,17 @@ The name of the test suite, which will be reflected in the results and related a
 ```
 ---
 
+### `testApp`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Set test application on suite level. See the full [usage](#testapp). If this field is not set, saucectl will pick the default `testApp` from [espresso](#espresso) level.
```
Comment (lysannep) [inline docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md:548]:
```suggestion
Sets the test application on the suite level. See the full [usage](#testapp). If this property is not set, `saucectl` will use the default `testApp` from the [`espresso`](#espresso) level.
```

File: docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md [hunk 3]
```diff
@@ -537,6 +537,17 @@ The name of the test suite, which will be reflected in the results and related a
 ```
 ---
 
+### `testApp`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Set test application on suite level. See the full [usage](#testapp). If this field is not set, saucectl will pick the default `testApp` from [xcuitest](#xcuitest) level.
```
Comment (lysannep) [inline docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md:543]:
```suggestion
Sets the test application on the suite level. See the full [usage](#testapp). If this property is not set, `saucectl` will use the default `testApp` from [`xcuitest`](#xcuitest) level.
```

**Review by lysannep — CHANGES_REQUESTED** [review]
Added a few minor suggestions. Thank you!

**Pull request merged**